index.html

```diff
@@ -25,7 +58,7 @@ <h1 class="header__h1">Web Development Global Conference 2023</h1>
         </div>
     </header>
     <main class="program">
-        <h2 class="subtitle__h2">Main Program</h2>
+        <h2 class="subtitle">Main Program</h2>
         <hr class="subtitle__line">
         <div class="card-container">
             <div class="program__card">
```

Comment:
Great job with using  semantic `<main>` element as wrapper here. Let's change this program cards container as `<article>` to make your HTML more semantic. It is important having an accessible and SEO friendly web pages.